snorkel/labeling/model/label_model.py

```diff
@@ -675,7 +675,7 @@ def _set_lr_scheduler(self) -> None:
                 total_steps - self.warmup_steps
             )
             lr_scheduler = optim.lr_scheduler.LambdaLR(  # type: ignore
-                self.optimizer, linear_decay_func
+                self.optimizer, lr_lambda=LRPolicy(self.warmup_steps, total_steps)
```

Comment:
Is linear_decay_func still being used elsewhere or should it be dropped form that file altogether?